softlayer/common/softlayer_helper.go

```diff
@@ -20,12 +20,7 @@ var (
 )
 
 func AttachEphemeralDiskToVirtualGuest(softLayerClient sl.Client, virtualGuestId int, diskSize int, logger boshlog.Logger) error {
```

Comment:
Since it is a change related with existing function, just implementation approach changed and the test cases had been changed accordingly, so no additional test cases needed.